internal/buildsign/pod/manager.go

```diff
@@ -120,7 +120,11 @@ func (pm *podManager) Sync(ctx context.Context, mld *api.ModuleLoaderData, pushI
 	return nil
 }
 
-func (pm *podManager) GarbageCollect(ctx context.Context, name, namespace, podType string, owner metav1.Object) ([]string, error) {
+func (pm *podManager) GarbageCollect(ctx context.Context, name, namespace string, action kmmv1beta1.BuildOrSignAction, owner metav1.Object) ([]string, error) {
+	podType := PodTypeBuild
+	if action == kmmv1beta1.SignImage {
+		podType = PodTypeSign
+	}
```

Comment:
Why do we need a translation, why can't we just add the action as the label instead of having another label for the pod type (which will always be the same as the action)?

Reply:
it means changing a lot of code that right now we inherit as is. We can do it later, but for now i would prefer a simple translation